src/core/Akka/Util/Try.cs

```diff
@@ -39,7 +41,7 @@ public static implicit operator Try<T>(T value)
         /// <summary>
         /// Shows if this is Success
         /// </summary>
-        public bool IsSuccess => Success.HasValue;
+        public bool IsSuccess { get; }
```

Comment:
Fix behavior difference between scala and our `Try<T>` implementation. In scala `Try`, a `Try` can have a null success value.

Reply:
If `T?` is nullable so could we - depends on if this gets used outside of Akka.Streams

Reply:
not with the current `Try<T>` implementation. In our implementation, we check to see if a `Try` is successful by checking to see if the value is null or not.

Passing a null value to the "success" constructor actualy creates a "failed" `Try` with null exception

Reply:
Got it